paddle/framework/tensor.h

```diff
@@ -37,6 +39,13 @@ class Tensor {
   }
 
   template <typename T>
+  T* raw_data() const {
```

Comment:
Since we always use `raw_data` in OpKernel. The mutable_data is only used for allocation. Maybe we could change `mutable_data` to `allocation`